iModelCore/GeomLibs/serialization/src/FlatBuffer/FixedStructs.cpp

```diff
@@ -2062,69 +2062,39 @@ PtrType BytesToXXX(Byte const *buffer, size_t const bufferSize, bool applyValida
     return nullptr;
     }
 
-IGeometryPtr BentleyGeometryFlatBuffer::BytesToGeometry(Byte const *buffer, size_t const bufferSize, bool applyValidation)
+IGeometryPtr BentleyGeometryFlatBuffer::BytesToGeometrySafe(Byte const *buffer, size_t bufferSize, bool applyValidation)
     {
-    return BytesToXXX<IGeometryPtr>(buffer, bufferSize, applyValidation, FBReader::ReadGeometry);
+    return BytesToXXXSafe<IGeometryPtr>(buffer, bufferSize, applyValidation, FBReader::ReadGeometry);
     }
 
-IGeometryPtr BytesToGeometrySafe(Byte const *buffer, size_t const bufferSize, bool applyValidation)
+IGeometryPtr BentleyGeometryFlatBuffer::BytesToGeometry(bvector<Byte> &buffer, bool applyValidation)
```

Comment:
If a const arg can be passed in, it should be const in the signature.

```suggestion
IGeometryPtr BentleyGeometryFlatBuffer::BytesToGeometry(bvector<Byte> const& buffer, bool applyValidation)
```